synapse/lib/cell.py

```diff
@@ -61,33 +61,36 @@ class CellApi(s_base.Base):
         sess = self.link.get('sess')  # type: s_daemon.Sess
         sess.user = user
 
-    async def allowed(self, *path):
+    # FIXME: why is this async?
```

Comment:
so it can be called by a remote caller with the same API call pattern as local.  ( also, any *non* async function is way less efficient over telepath because it assumes it has to fire through an executor )